src/main/java/org/jabref/gui/filelist/LinkedFilesEditDialogViewModel.java

```diff
@@ -119,7 +119,7 @@ public StringProperty descriptionProperty() {
     }
 
     public LinkedFile getNewLinkedFile() {
-        return new LinkedFile(description.getValue(), link.getValue(), selectedExternalFileType.getValue().toString());
+        return new LinkedFile(description.getValue(), link.getValue(), selectedExternalFileType.getValue() == null ? null : selectedExternalFileType.getValue().toString());
```

Comment:
As we wanted to get ride of null's, I don't like this change. Maybe we should use a `MonadicBinding` (optional binding) and use an empty string as file type?